Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -425,8 +463,7 @@ extension ActivityStreamPanel: DataObserverDelegate {
     }
 
     func getHighlights() -> Success {
-        let num = Int(Section.highlights.numberOfItemsForRow())
-        return self.profile.recommendations.getHighlights().both(self.profile.recommendations.getRecentBookmarks(num)).bindQueue(.main) { (highlights, bookmarks) in
+        return self.profile.recommendations.getHighlights().both(self.profile.recommendations.getRecentBookmarks(2)).bindQueue(.main) { (highlights, bookmarks) in
```

Comment:
This seems confusing that the `2` is hard-coded without explanation. If we don't need the value for `Section.highlights.numberOfItemsForRow()` anymore, then maybe just add a comment explaining why.